src/lib/routes/frontsite/Frontsite.js

```diff
@@ -28,59 +29,67 @@ export default class FrontSite extends Component {
   render() {
     const staticPagesRoutes = getStaticPagesRoutes();
     return (
-      <div className="frontsite">
-        <ILSMenu />
-        <Notifications className="compact" />
-        <Container fluid className="fs-content">
-          <Banner />
-          <Switch>
-            <Route
-              exact
-              path={FrontSiteRoutes.home}
-              render={props => <Home {...props} {...this.props} />}
-            />
-            <Route
-              exact
-              path={FrontSiteRoutes.documentsList}
-              component={LiteratureSearch}
-            />
-            <Route
-              exact
-              path={FrontSiteRoutes.documentDetails}
-              component={DocumentDetails}
-            />
-            <Route
-              exact
-              path={FrontSiteRoutes.seriesDetails}
-              component={SeriesDetails}
-            />
-            <Route
-              exact
-              path={FrontSiteRoutes.documentRequestForm}
-              component={DocumentRequestForm}
-            />
-            <Route
-              exact
-              path={FrontSiteRoutes.patronProfile}
-              component={PatronProfile}
-            />
-            <Route
-              exact
-              path={FrontSiteRoutes.openingHours}
-              component={OpeningHours}
-            />
-            <Route exact path={FrontSiteRoutes.errors} component={ErrorsPage} />
-            {staticPagesRoutes.map(route => (
-              <Route key={route} exact path={route} component={StaticPage} />
-            ))}
-            {this.renderCustomStaticPages()}
-            <Route>
-              <NotFound />
-            </Route>
-          </Switch>
-        </Container>
-        <ILSFooter />
-      </div>
+      <Overridable id="Frontsite.layout" {...this.props}>
```

Comment:
I'm not sure if it is needed to overwrite all the routes at this point
@ntarocco @topless @jrcastro2 @FlorianCassayre what do you guys think?

Reply:
The biggest issue I see here is that we need to think that Overridable is a black box: when overriding it, we cannot access to the outside props/vars/imports.
So we should pass any extra props, e.g. the `staticPagesRoutes`.

We should probably take the habit to extract the inner part in another cmp, even if we could still do this mistake:

```
<Overridable ... staticPagesRoutes>
  <NewCmp props... staticPagesRoutes>
</Overridable>

class NewCmp ...
    staticPagesRoutes
```